test/relations.test.js

```diff
@@ -835,6 +835,23 @@ describe('relations', function() {
         });
       });
 
+      context('findById with filter include that contains scope with string filter fields', function() {
+        it('should be accepted and convert fields to array.', function(done) {
+          var includeFilter = {include: {relation: 'patients', scope: {fields: 'name'}}};
+          var physicianId = physician.id;
+          Physician.findById(physicianId, includeFilter, function(err, ch) {
```

Comment:
please call it `result` instead of `ch` in the callback and replace all the subsequent `ch` with `result`